WordPress/wordpress-playground
# Route requests for missing static files using remote asset metadata

## What is this PR doing?

This PR:
- Includes a list of remote asset paths with minified WP builds
- Updates request handling to only handle missing files as static when they are explicitly listed as remote assets
(perhaps this should be split into multiple PRs)
- Updates request routing to behave more like a regular web server

Related to #1365 in that we need better information to judge whether to handle a request for a missing static file as PHP.

## What problem is it solving?

Prior to this PR, we could not easily tell whether we should request a missing static asset from the web or delegate the request to PHP.

## How is the problem addressed?

By including a list of remote asset paths with minified WP builds, we can judge which missing static file paths represent remote assets and which can be delegated to PHP.

## Testing Instructions

- CI tests
- Examine a minified WP zip and the contents of its playground-remote-asset-paths file
- Test manually with `npm run dev` and observe that Playground loads normally with no unexpected errors in the console
